pkg/controller/restore_controller.go

```diff
@@ -492,6 +535,8 @@ func (c *restoreController) runRestore(restore *api.Restore, bucket string) (res
 	}
 	defer c.pluginManager.CloseRestoreItemActions(restore.Name)
 
+	// Any return statement above this line means a total restore failure
+	// Some failures after this line *may* be a total restore failure
 	logContext.Info("starting restore")
```

Comment:
not related directly to this PR but our logging is very redundant, since we're also logging start/complete of restores in the calling function